src/backend/marsha/core/serializers/playlist.py

```diff
@@ -26,15 +28,43 @@ class Meta:
             "users",
         ]
 
+    portable_to = serializers.SerializerMethodField(read_only=False)
+
+    def get_portable_to(self, obj):
+        """Getter for portable_to attribute instead of PlaylistSerializer to prevent recursion."""
+        return [
+            {"id": playlist.id, "title": playlist.title}
+            for playlist in obj.portable_to.all()
+        ]
+
     def create(self, validated_data):
         """Create the Playlist object with the passed data."""
         return Playlist.objects.create(**validated_data)
 
+    def update(self, instance, validated_data):
+        """
+        Update the Playlist object with the passed data.
+
+        Instance id is removed from portable_to attribute.
+        """
+        if "portable_to" in validated_data and str(instance.id) in validated_data.get(
+            "portable_to"
+        ):
+            validated_data.get("portable_to").remove(str(instance.id))
+        return super().update(instance, validated_data)
+
+    def to_internal_value(self, data):
+        """Override to add portable_to playlists."""
+        value: OrderedDict = super().to_internal_value(data)
```

Comment:
If I understand well you're turning the readonly function field in a writable field? Why not make the field writable the DRF way? If it's not possible for some reason, you should at least make some validation?

Reply:
I'm not sure to understand what you mean by `turning the readonly function field in a writable field`.
Yes, by default a SerializerMethodField is read only, but that's all :thinking: 
But surely I'm missing something here…

Reply:
your docstring says `"""Override to add portable_to playlists."""`. So I understand you are trying to write to the `portable_to` field which otherwise is readonly. 
So I think you can't do it just like that but need proper validation of the values submitted.

Reply:
Oh ok, I see.
I'll search for a better way.